src/index.html

```diff
@@ -7,8 +7,40 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+
+      <a href=" ">
```

Comment:
there shouldn’t be empty line between child and parent element
```suggestion
    <header class="header">
      <a href=" ">
```

Reply:
there shouldn’t be empty line between child and parent element